commands/register.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
 		if(!args[0]) return bot.cmds.help.execute(bot, msg, ["register"], cfg);
 
 		//check arguments
-		let brackets = msg.content.slice(msg.content.indexOf(args[0])+args[0].length+1).trim().split("text");
+		let content = args.join(" ");
```

Comment:
Using `args.join(" ")` here means that if people try to register using any of the "command grouping" symbols (like `""`, `''`, etc) it will strip them and give them empty brackets. I think this needs to set `content` to the raw message content with the command part sliced off instead.